apps/web/src/pages/projects/[project]/areas/create.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react'
+import { zodResolver } from '@hookform/resolvers/zod'
+import * as z from 'zod'
+import { useForm } from 'react-hook-form'
+
+import { Button } from "@/components/ui/button"
+import {
+    Form,
+    FormControl,
+    FormField,
+    FormItem,
+    FormLabel,
+    FormMessage
+} from "@/components/ui/form"
+import { Input } from "@/components/ui/input"
+import { Textarea } from "@/components/ui/textarea"
+import { PageLayout} from "@/components/ui/page-layout"
+import { Heading } from '@/components/ui/typography'
+import { Separator } from '@/components/ui/separator'
+import { useRouter } from 'next/router'
+
+const formSchema = z.object({
+    name: z.string(),
+    polygon: z.string()
+})
+
+export default function ProjectAreaCreate() {
+    const router = useRouter();
+    const projectId = router.query.project;
+    const exampleText = '[{"lat":52.08526843203928,"lng":4.273874759674072},{"lat":52.086283702713075,"lng":4.276385307312012}, ...]'
+    
+    const form = useForm<z.infer<typeof formSchema>>({
+        resolver: zodResolver<any>(formSchema),
+        defaultValues: {
+        }
+    })
+
+    function onSubmit(values: z.infer<typeof formSchema>) {
+        createArea(`/api/openstad/api/project/${projectId}/area`,
```

Comment:
Ik denk dat het handiger is om dit binnen een hook te plaatsen. Dus hetzelfde ongeveer als voor de widget config. Doordat je een fetch doet zonder swr wordt de cache niet direct getriggered en kan het enkele seconden duren voordat je de nieuwste data hebt

Reply:
Maar willen we dan voor elke mogelijke onderdeel dezelfde hook gaan schrijven? Het lijkt mij redelijk omslachtig in dat aspect, maar ik weet dan ook niet alle specificaties eromheen